README.md

```diff
@@ -28,7 +28,7 @@ in WeeChat.
 
 #Support
 
-IRC channel: #weechat-otr on Freenode.
+IRC channel: [#weechat-otr on Freenode](ircs://chat.freenode.net:+6697).
```

Comment:
This is the form that I am usually seeing, but I can change it next time I have access to git.